src/index.ts

```diff
@@ -67,6 +67,14 @@ let data = [];
 const testData = ['Person 1', 'Person 2', 'Person 3', 'Person 4'];
 
 let imageUrlForSpinner = '';
+const imageUrlInput = document.getElementById('imageUrlInput') as HTMLInputElement;
+
+function imageUrlSetter() {
+  imageUrlForSpinner = imageUrlInput.value;
```

Comment:
In its current form, users are able to copy and paste a full URL, including the querystring, to share a customised selection wheel. It would be great for this function to also add the querystring for the image URL that the user has selected. 

Elsewhere in this file, we have this code: 

```ts
const urlParams = new URLSearchParams(window.location.search);
if (urlParams.has('imgUrl')) {
  imageUrlForSpinner = urlParams.get('imgUrl');
}
```

This enables us to get the imgUrl parameter from the querystring. 

Here, you're getting the url from the input field, so you could write some similar code within this function to set the querystring imgUrl parameter.

Reply:
It would also be nice to populate the `imageUrlInput` text input field with the url from the querystring, if there is one. This could be done when we run `imageUrlForSpinner = urlParams.get('imgUrl');`

Reply:
I wasn't exactly sure how to go about this so I had the first function load a new window.location.href, and the second function is left as originally written. A notification that the page would reload seems appropriate, would you like me to add that below the input field? Everything requested is done. 😄 

Also sorry if the commit > tag stuff still isn't right (when I do it), I'm using fork and though I know how I'm not sure what'll come out of it.

Reply:
Don't worry about changing the `window.location` - just updating the querystring is sufficient as people can copy it from their URL bar in the browser, if they want to share it, so no notification about reloading is necessary 🙂

No worries - just have a go!

Reply:
Howdy, I made the requested changes. Thank you again for putting something like this out there.  I've absolutely loathed git since I started learning to program and my interaction with you is probably the most useful engagement with it I've had.

Reply:
Almost there now! This version of the code will do two undesirable things:

1. Reload the page
2. Add an image url to the end of any existing image urls

The second thing is particularly bad, as it introduces a bug. A user can set a URL, then change it, and then two URLs will show in the querystring. 
<img width="1029" alt="image" src="https://github.com/jcreek/SelectionWheel/assets/8179928/9b2e05c1-598b-442f-9423-8f9b00b0edb1">

Instead, I'd recommend using something like the below. I haven't checked my syntax so you'll need to make sure it's right, but this should either modify the imgUrl or add it if there isn't one, without any duplicates and without reloading the page 🙂

```ts
const urlParams = new URLSearchParams(window.location.search);
urlParams.set(imageUrlForSpinner);
```